app/rot-carrier/app.toml

```diff
@@ -137,37 +139,22 @@ pins = [
 # SCK
 { pin =  { port = 0, pin = 6 }, alt = 1 },
 # CS, not strictly necessary but handy for debugging
-{ pin = {port = 0, pin = 20}, alt =  1},
+# { pin = {port = 0, pin = 20}, alt =  1},
 ]
 spi_num = 3
 
-[tasks.ping]
-name = "task-ping"
-features = ["uart"]
-priority = 6
-max-sizes = {flash = 8192, ram = 2048}
-start = true
-task-slots = [{peer = "pong"}, "usart_driver"]
-
-[tasks.pong]
-name = "task-pong"
-priority = 5
-max-sizes = {flash = 8192, ram = 1024}
-start = true
-task-slots = ["user_leds"]
-
```

Comment:
Did those get removed intentionally? It's nice to see the comforting blinking lights when testing on the carrier if we still have space.

Reply:
I'll look. At one time I was concerned about the crashing one interfering with real time stuff.

Reply:
I put them back in with priority changes to keep them from preempting anything else.